ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -239,3 +241,46 @@ def test_tensorflow_pinv(
         validate_args=False,
         name=None,
     )
+
+
+# tensordot
+@handle_cmd_line_args
+@given(
+    dtype_x_y_axes=_get_dtype_value1_value2_axis_for_tensordot(
+        available_dtypes=helpers.get_dtypes("numeric"),
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.tensordot"
+    ),
+)
+def test_tensorflow_tensordot(
+    *,
+    dtype_x_y_axes,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    (
+        dtype,
+        x,
+        y,
+        axes,
+    ) = dtype_x_y_axes
+
+    as_variable = [as_variable, as_variable]
+    native_array = [native_array, native_array]
+
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="tensorflow",
+        fn_tree="tensordot",
+        x=x,
```

Comment:
You have to pass this as a numpy array with the dtype which you are unpacking, kindly refer to this in the docs if you have any confusion. https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends_tests.html